variable_distance_exploration/lib/experimental_media_query.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+/// Interaction models.
+enum DeviceCorporealContext {
+  /// Viewed only.
+  ambient,
+
+  /// Interactive.
+  focused,
+
+  /// Mixed: sometimes viewed only, sometimes interactive.
+  multi,
+}
+
+/// Real world device purposes.
+enum DeviceCorporealType {
+  /// In-car console display.
+  auto,
+
+  /// Stationary / in-home tablet on a surface.
+  countertop,
+
+  /// Traditional desktop computer.
+  desktop,
+
+  /// Foldable phone.
+  foldableHandheld,
+
+  /// Slab-style phone.
+  slabHandheld,
+
+  /// Mobile tablet.
+  tabletHandheld,
```

Comment:
I don't know much about the conversation going on here, but it seems reasonable to me to maybe consider `tabletHandheld`s to just be the same as `slabHandheld`s, just differing in size. They seem to have the same general form.

Just a thought.

Reply:
They differ in layout expectations. Landscape tablet tends to have a split screen.